### PR TITLE
Update notices.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Added Admin Advisor content.

### CODE REVIEW

File: source/administration/notices.rst [hunk 1]
```diff
@@ -15,13 +15,28 @@ Administrator notices are used to inform System Admins when a new server version
...
+
+When these notifications are triggered, a System Admin receives a bot message notification and, for larger instances, a dismissable daily banner with quick access to contact support. The triggers include the number of registered users in your deployment reaching a predefined threshold, the number of posts made, and the number of channels you've created. For more information, you can view the source code.
+
+These notifications also give System Admins quick access to optionally contact support for additional assistance with a single click as well as links to useful resources for user management and provisioning to support large systems.
+
+Selecting **Contact us** in the Admin Advisor banner dismisses the banner notification. In addition, information such as the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number are sent to us. This information is used to contact you as requested and to help us better understand your needs.
```
Comment (jasonblais) [inline source/administration/notices.rst:27]:
```suggestion
Selecting **Contact us** in the Admin Advisor banner dismisses the banner notification. In addition, contact information such as the email address and name associated with your Mattermost account is sent to us, so we can contact you as requested and to help us better understand your needs. The number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number are also sent to us to better understand your environment and hence provide more tailored support.
```

File: source/administration/notices.rst [hunk 2]
```diff
@@ -15,13 +15,28 @@ Administrator notices are used to inform System Admins when a new server version
 
 Administrator notices can be disabled in the **System Console > Notices** page.
 
+Admin Advisor notices
+^^^^^^^^^^^^^^^^^^^^^
+
+Admin Advisor notifications are included in Mattermost from v5.26. These notifications are designed to provide guidance and assist in choosing features to support your deployment as it scales.
+
+When these notifications are triggered, a System Admin receives a bot message notification and, for larger instances, a dismissable daily banner with quick access to contact support. The triggers include the number of registered users in your deployment reaching a predefined threshold, the number of posts made, and the number of channels you've created. For more information, you can view the source code.
```
Comment (jasonblais) [inline source/administration/notices.rst:23]:
```suggestion
When these notifications are triggered, a System Admin receives a bot message notification and, for larger instances, a dismissable daily banner with quick access to contact support. The notifications are triggered based on usage such as the the number of registered users, posts or channels in your deployment reaching a predefined threshold.
```

File: source/administration/notices.rst (hunk 1 above)
Comment (jasonblais) [reply source/administration/notices.rst:27]: ^the wording isn't the best, but the intention with the changes is to make it clear right away that the information sent to us is used to allow us contact them, and the additional information allows us to offer more tailored/custom support.

**Review by jasonblais — CHANGES_REQUESTED** [review]
Minor suggestions, thanks Justine!

File: source/administration/notices.rst [hunk 3]
```diff
@@ -15,13 +15,33 @@ Administrator notices are used to inform System Admins when a new server version
...
+When these notifications are triggered, a System Admin receives a bot message notification and, for larger instances, a dismissable daily banner with quick access to contact support. The notifications are triggered based on usage such as the the number of registered users, posts or channels in your deployment reaching a predefined threshold.
+
+These notifications also give System Admins quick access to optionally contact support for additional assistance as well as links to useful resources for user management and provisioning to support large systems.
+
+Managing admin advisor notifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Admin Advisor notifications are run on a schedule based on specific triggers.
+
+- Selecting **Contact us** in the Admin Advisor banner dismisses the banner notification. In addition, information such as the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number are sent to us. This information is used to contact you as requested and to help us better understand your needs.
```
Comment (justinegeffen) [inline source/administration/notices.rst:32]:
```suggestion
- Selecting **Contact us** in the Admin Advisor banner dismisses the banner notification. In addition, information such as the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number are sent to us. We use this information to help us better understand your requirements before we contact you.
```

Comment (justinegeffen) [reply source/administration/notices.rst:32]: Slight rewording for this, to emphasize how we use the data.

Comment (jasonblais) [reply source/administration/notices.rst:32]:
```suggestion
- Selecting **Contact us** in the Admin Advisor banner or the bot message contacts our support team, and dismisses the notification. No further Admin Advisor notifications are sent thereafter. In addition, information such as the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number are sent to us. We use this information to help us better understand your requirements before we contact you.
```

Yes this is better. Slight adjustments above.

File: source/administration/notices.rst [hunk 4]
```diff
@@ -15,13 +15,33 @@ Administrator notices are used to inform System Admins when a new server version
...
+When these notifications are triggered, a System Admin receives a bot message notification and, for larger instances, a dismissable daily banner with quick access to contact support. The notifications are triggered based on usage such as the the number of registered users, posts or channels in your deployment reaching a predefined threshold.
+
+These notifications also give System Admins quick access to optionally contact support for additional assistance as well as links to useful resources for user management and provisioning to support large systems.
+
+Managing admin advisor notifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Admin Advisor notifications are run on a schedule based on specific triggers.
+
+- Selecting **Contact us** in the Admin Advisor banner dismisses the banner notification. In addition, information such as the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number are sent to us. We use this information to help us better understand your requirements before we contact you.
```
Comment (justinegeffen) [inline source/administration/notices.rst:32]:
```suggestion
Selecting **Contact us** in the Admin Advisor banner dismisses the banner notification. In addition, contact information such as the email address and name associated with your Mattermost account is sent to us, so we can contact you as requested and to help us better understand your needs. The number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number are also sent to us to better understand your environment and hence provide more tailored support.
```

**Comment by amyblais** [comment]
The formatting of the page looks broken, possibly due to the header formatting.

**Review by jasonblais — APPROVED** [review]

**Comment by justinegeffen** [comment]
> The formatting of the page looks broken, possibly due to the header formatting.

I see that it's not rendering the diff but it does look like it's working on preview. I just fixed the list formatting and will check again when the preview is up.

**Comment by justinegeffen** [comment]
@amyblais, confirmed as displaying in preview. Not sure why it's not rendering here.

Screen Shot 2020-11-16 at 14 38 36

File: source/administration/notices.rst [hunk 5]
```diff
@@ -15,13 +15,33 @@ Administrator notices are used to inform System Admins when a new server version
...
+---------------------
+
+Admin Advisor notifications are included in Mattermost from v5.26. These notifications are designed to provide guidance and assist in choosing features to support your deployment as it scales.
+
+When these notifications are triggered, System Admins receive a bot message notification and, for larger instances, a dismissable daily banner with quick access to contact support. The notifications are triggered based on usage such as the the number of registered users, posts, or channels in your deployment reaching a predefined threshold.
+
+These notifications also give System Admins quick access to optionally contact support for additional assistance as well as links to useful resources for user management and provisioning to support large systems.
+
+Managing admin advisor notifications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (amyblais) [inline source/administration/notices.rst:28]:
```suggestion
---------------------
```

**Review by amyblais — APPROVED** [review]

**Pull request merged**